torch/nn/quantized/_reference/modules/conv.py

```diff
@@ -0,0 +1,136 @@
+import torch
+import torch.nn.quantized as nnq
+import torch.nn.functional as F
+from typing import Optional
+from torch.nn.common_types import _size_1_t
+from torch.nn.modules.utils import _single
+
+class _ConvNd(nnq._ConvNd):
+    """ A reference version of nn.quantized.Conv2d
+        we will not pack the parameters in this module, since weight packing is an
+        optimization for quantized backends supported in PyTorch (fbgemm/qnnpack),
+        this is useful when user want to use this module in other backends like Glow.
+    """
+    __annotations__ = {"_bias": Optional[torch.Tensor]}
+
+    def _save_to_state_dict(self, destination, prefix, keep_vars):
+        super()._save_to_state_dict(destination, prefix, keep_vars)
+        destination[prefix + '_qweight'] = self._qweight
+        destination[prefix + '_bias'] = self._bias
+
+    def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
+                              missing_keys, unexpected_keys, error_msgs):
+        self._qweight = state_dict[prefix + '_qweight']
+        self._bias = state_dict[prefix + '_bias']
+        state_dict.pop(prefix + '_qweight')
+        state_dict.pop(prefix + '_bias')
+
+        super()._load_from_state_dict(
+            state_dict, prefix, local_metadata, False,
+            missing_keys, unexpected_keys, error_msgs)
+
+    def _weight_bias(self):
+        return self._qweight, self._bias
+
+    def set_weight_bias(self, w: torch.Tensor, b: Optional[torch.Tensor]) -> None:
+        self._qweight = w
+        self._bias = b
```

Comment:
I guess these ones are not reusable because the weight is stored unpacked?

Reply:
yes, that's correct